scripts/startup.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Copyright (C) 2018 Expedia Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+SHUNTING_YARD_HOME=/opt/shunting-yard
+
+[[ ! -z $SHUNTINGYARD_CONFIG_YAML ]] && echo "$SHUNTINGYARD_CONFIG_YAML"|base64 -d > ${SHUNTING_YARD_HOME}/conf/shunting-yard-config.yml
```

Comment:
These variables are environment variables which will be set by my initialization code in Apiary-replication.

Looks like this:

``` ],
    "environment":[
     {
        "name": "HEAPSIZE",
        "value": "${heapsize}"
     },
     {
        "name": "SHUNTINGYARD_CONFIG_YAML",
        "value": "${shuntingyard_config_yaml}"
     }
    ]```